js/events/core/events_engine.js

```diff
@@ -171,14 +171,22 @@ var getHandlersController = function(element, eventName) {
 
             var firstHandlerForTheType = eventData.handleObjects.length === 1;
             var shouldAddNativeListener = firstHandlerForTheType && eventNameIsDefined;
+            var options;
```

Comment:
I suppose it's too common name for such a big method. Maybe `nativeListenerOptions`?